RedHat_NLP/Deliverable 1.ipynb

```diff
@@ -0,0 +1,285 @@
+            "text/plain": [
```

Comment:
You should add a markdown cell below this graph explaining what it tells us about our data.